### ISSUE TITLE
how to delete row in nayjest grid

### ISSUE THREAD

**Issue by DevGentle** [Nayjest/Grids#156]
Hi me again 

I've got stuck about delete data row in grid

This is my code delete method in second of <li>

<img width="716" alt="screen shot 2559-09-06 at 10 21 31 pm" src="https://cloud.githubusercontent.com/assets/20085539/18279604/55419d80-7480-11e6-9de4-b9b3299ba788.png">

and this is a result 

<img width="1045" alt="screen shot 2559-09-06 at 10 24 20 pm" src="https://cloud.githubusercontent.com/assets/20085539/18279723/bb1c18a6-7480-11e6-98b9-9eeea3c28ae5.png">

That's all of my question. I really don't know how to do next plz help me again.

Thanks.

**Comment by wdog** [comment]
**Nesting Forms is not allowed in html5 see [4.10.3](https://www.w3.org/TR/html5/forms.html#the-form-element)**

>Content model:
>   Flow content, but with no form element descendants.

personally I create a form on the fly using JS

steps: 

-(**buttons**) inside the Grid, in the callback for the buttons,  I simply create a link to the DELETE route  es. `route('user.destroy', $id) `  passing an attribute `data-delete=""` 

```html
<a data-delete="" href="https://localhost/user/7">Remove User</a>
```

-(**view**) in the view I include a javascript file and that's all -> this is my version of the file [deleter.blade.js](https://gist.github.com/wdog/885ee8473cf96cd1bd7fbd336e6fec45#file-deleter-blade-js)

Basically the scripts creates a form in the body for ever link having the attribute `data-delete=""`

note 1: 
I use the swal package to show a nice alert (https://limonte.github.io/sweetalert2/)

note 2:
I cannot remender where I took the idea or the initial script sorry :cry: 

note 3:
the form is has a DELETE method
```js
var hiddenInput =
          $('<input>', {
            'name': '_method',
            'type': 'hidden',
            'value': 'DELETE'
          });
```

Let me know if you have any doubts , I know I'm a bit messy

**Comment by wdog** [comment]
I forgot to say that the problem is that the grid is inside a form (for filters )

*IDEA!* :bulb: 

I can get out the form from the grid like this:

```html
<form id='frm_{table_id}'></form>

<table id='{table_id}'>
...input fields relative to the form filters


... other forms not related  to the form filters
</table>
```

and than use ( **only for html5** ) the `form` in the input fields 
```html
<input type='text' form='{table_id}' />
```
@DevGentle  try this technique in your form maybe is enough

here's an [example](https://www.impressivewebs.com/demo-files/html5-form-attribute/) of the worst case

**Comment by DevGentle** [comment]
Hi @wdog thanks a lot.

**Comment by JosueMorales** [comment]
Pretty clean and nice solution! @wdog
